### DISCUSSION TITLE
How to move the container image storage to the SD card on a Steam Deck?

### DISCUSSION THREAD

**Discussion by herzenschein** [containers/podman#22298] [Q&A]
Hi, I'd like to move my rootless podman images from my Steam Deck to an external SD card (/run/media/mmcblk0p1/podman/). I want to use it with latest podman.

Context about the Steam Deck: it's an immutable distro without sudo or root. Its default distrobox is too old, so I installed 1.7.1.0; the podman version is 4.5.1.

I am aware of both https://github.com/containers/podman/issues/20068 and https://github.com/containers/podman/issues/1916. I tried the following:

1. Running `podman system reset`

2. Creating a ~/.config/containers/storage.conf with the following contents:

```ini
[storage]
driver = "overlay"
graphroot = "/run/media/mmcblk0p1/podman"
rootless_storage_path = "/run/media/mmcblk0p1/podman"
```

It works for pulling images.

3. Running `distrobox create --image <anyimage>` which creates a new container

4. Running `distrobox enter <anyimagename>` to enter said container

Which fails with `private: OCI permission denied`. It is my understanding that this is a podman issue, not a distrobox issue. I'm asking this here because I found no proper documentation for this, only the aforementioned issues, so it's possible that I'm just missing something in my config file.

**Comment by rhatdan** [comment]
If this is an SELinux system, it is most likely SELinux settings.

https://github.com/containers/storage/blob/main/docs/containers-storage.conf.5.md#storage-table
